Fix update-upgrade-policy-state script (#211)
openshift/managed-scripts: scripts/SREP/update-upgrade-policy-state/README.md

## Usage

> To cancel the upgrade policy of the current cluster, you can run the following commands for prod and staging clsuters:

Reviewer: ```suggestion
To cancel the upgrade policy of the current cluster, you can run the following commands for prod and staging clusters:
```

> For production clusters (no need to specify env=prod as parameter since it is prod by default) : 

Reviewer: ```suggestion
Note : For production clusters (specifying `$env` prod is not needed since prod is the default setting.) : 
```

```bash
ocm backplane managedjob create SREP/update-upgrade-policy-state
```

> For staging clusters (need to specify env=stage as parameter): 

Reviewer: ```suggestion
For staging clusters (environment parameter has to be specified as env=stage ): 
```
